packages/config/package.json

```diff
@@ -16,25 +16,9 @@
     "./register": "./dist/deprecated/register.js",
     "./node": "./dist/deprecated/node.js"
   },
-  "typesVersions": {
-    "*": {
-      "index": [
-        "./src/exports/index.ts"
-      ],
-      "internal": [
-        "./src/exports/internal.ts"
-      ],
-      "library": [
-        "./src/deprecated/library/index.ts"
-      ],
-      "register": [
-        "./src/deprecated/register/index.ts"
-      ],
-      "node": [
-        "./src/deprecated/node/index.ts"
-      ]
-    }
-  },
+  "files": [
+    "dist"
+  ],
```

Comment:
just noting my own learnings here:
we don't need to include everything from `.npmignore` here because by default, npm picks up `package.json` as well as `README` (with any extension): https://docs.npmjs.com/cli/v10/configuring-npm/package-json#files